### PR TITLE
[FRONT-78] Add time selector to details page

### PR DESCRIPTION
**Pull request by WrappedSuperClass**
### Description of issue
Time Selector: 

Upon clicking "Edit Timeframe", there should be a dropdown with "Today/This week/This month" options. 

### How I implemented
I reused the Modal.tsx and added it to the code using the three requried buttons. 
To fix the modal under the button while scrolling up und down i changed the css class from fixed to absolute.

### CODE REVIEW

File: src/pages/details.tsx [hunk 1]
```diff
@@ -41,13 +55,30 @@ const DetailPage = () => (
...
+                  variant="noBorderNoBG"
+                  text="This Week"
+                  fullWidth
+                  onClick={() => handleClick}
+                />
+                <Button
+                  variant="noBorderNoBG"
+                  text="This Month"
+                  fullWidth
+                  onClick={() => handleClick}
```
Comment (pczern) [inline src/pages/details.tsx:78]: handleClick

File: src/pages/details.tsx [hunk 2]
```diff
@@ -41,13 +55,30 @@ const DetailPage = () => (
...
+                Icon={Calendar}
+                order="ltr"
+              />
+              <Modal isOpen={isModalOpen} onClose={handleCloseModal}>
+                <Button variant="noBorderNoBG" text="Today" fullWidth onClick={() => handleClick} />
+                <Button
+                  variant="noBorderNoBG"
+                  text="This Week"
+                  fullWidth
+                  onClick={() => handleClick}
```
Comment (pczern) [inline src/pages/details.tsx:72]: handleClick

**Review by pczern — CHANGES_REQUESTED** [review]

File: src/pages/details.tsx [hunk 3]
```diff
@@ -15,6 +15,8 @@ import withAuth from '@/components/side-effects/withAuth'
 import SmallSidebar from '@/components/page/details/Sidebar'
 import ProjectInformation from '@/components/page/details/ProjectInformation'
 import { data, hackerNewsListMock, tweetListMock } from '@/data/detailPageMocks'
+import Modal from '@/components/pure/Modal'
+import { useState } from 'react'
 
 const handleClick = () => {
   // TODO: Implement click handler
```
Comment (cnk1dev) [inline src/pages/details.tsx:22]:
```suggestion
  // @TODO Implement click handler
```

**Review by cnk1dev — CHANGES_REQUESTED** [review]

File: src/pages/details.tsx (hunk 1 above)
Comment (cnk1dev) [reply src/pages/details.tsx]: fixed

File: src/pages/details.tsx (hunk 3 above)
Comment (cnk1dev) [reply src/pages/details.tsx]: fixed

**Review by cnk1dev — APPROVED** [review]

**Review by pczern — APPROVED** [review]

**Pull request merged**